sdk/data/cosmos/azcosmos/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/Azure/azure-sdk-for-go/sdk/cosmos/azcosmos
+module github.com/Azure/azure-sdk-for-go/sdk/data/cosmos/azcosmos
 
 go 1.13
```

Comment:
This should be go 1.16, which is our new min supported version

Reply:
Done